meson.build

```diff
@@ -35,6 +34,15 @@ c_args = [
     '-sym', 'on'
 ]
 
+if get_option('gdb_debugging')
+    c_args += [
+        '-O1,p',
+        '-inline', 'off'
```

Comment:
this is to help the debugger out a little, ideally this would be `-O0` but I had some issues with running the ROM with optimizations disabled completely